### PR TITLE
MGMT-15261: Add pkg/genisoimage unit-tests

### PR DESCRIPTION
**Pull request by nmagnezi**

### CODE REVIEW

File: pkg/genisoimage/genisoimage_test.go [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+		copyCmd, copyCmdrgs := executer.FormatCommand(fmt.Sprintf(genDataImageCmd, fakeCachePath, fakeImageName, fakeDataPath))
+		mockExecuter.EXPECT().Execute(copyCmd, copyCmdrgs).Return("", nil).Times(1)
+
+		skopeo := NewGenIsoImage(mockExecuter)
+		err := skopeo.GenerateImage(fakeCachePath, fakeImageName, fakeDataPath)
+		Expect(err).ToNot(HaveOccurred())
+	})
+
+	It("genisoimage NewGenIsoImage  - failure", func() {
```
Comment (danielerez) [inline pkg/genisoimage/genisoimage_test.go:42]: nit: redundant whitespace

File: pkg/genisoimage/genisoimage_test.go [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+		fakeDataPath  = "/path/to/data"
+		fakeImageName = "testdata.iso"
+	)
+
+	BeforeEach(func() {
+		ctrl = gomock.NewController(GinkgoT())
+		mockExecuter = executer.NewMockExecuter(ctrl)
+	})
+
+	It("genisoimage NewGenIsoImage - success", func() {
```
Comment (danielerez) [inline pkg/genisoimage/genisoimage_test.go:28]: Maybe replace NewGenIsoImage with GenerateImage, as that's the func we're actually testing.
E.g. GenerateImage - success

Comment (nmagnezi) [reply pkg/genisoimage/genisoimage_test.go]: Done

File: pkg/genisoimage/genisoimage_test.go (hunk 1 above)
Comment (nmagnezi) [reply pkg/genisoimage/genisoimage_test.go]: Done

**Pull request merged**